File: LearnProperty/LearnProperty/SecondViewController.m
//
//  SecondViewController.m
//  LearnProperty
//
//  Created by EthanLin on 2018/7/28.
//  Copyright © 2018 EthanLin. All rights reserved.
//

#import "SecondViewController.h"
#import "ViewController.h"

@interface SecondViewController ()

@end

@implementation SecondViewController


- (void)viewDidLoad {
    [super viewDidLoad];
    ViewController* firstVC = [[ViewController alloc] init];
//    [firstVC anotherName];
    NSLog(@"%@",firstVC.anotherName);
    // Do any additional setup after loading the view.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: Test/Test/SecondViewController.swift
//
//  SecondViewController.swift
//  Test
//
//  Created by EthanLin on 2018/7/28.
//  Copyright © 2018 EthanLin. All rights reserved.
//

import UIKit

class SecondViewController: UIViewController {

    @IBOutlet weak var labek: UILabel!
    override func viewDidLoad() {
        super.viewDidLoad()
        let firstVC = ViewController()
        firstVC.anotherName = "No"
        labek.text = firstVC.anotherName
        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
